docs/integrations/language-clients/csharp.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_label: 'C#'
+sidebar_position: 6
+keywords: ['clickhouse', 'cs', 'c#', '.net', 'dotnet', 'csharp', 'client', 'driver', 'connect', 'integrate']
+slug: /integrations/csharp
+description: 'The official C# client for connecting to ClickHouse.'
+title: 'ClickHouse C# Driver'
+---
+
+# ClickHouse C# Client
+
+The official C# client for connecting to ClickHouse. 
+The client source code is available in the [GitHub repository](https://github.com/ClickHouse/clickhouse-cs).
+Originally developed by [Oleg V. Kozlyuk](https://github.com/DarkWanderer).
+
+## Supported .NET Versions {#supported-net-versions}
+
+`ClickHouse.Driver` supports the following .NET versions:
+* .NET Framework 4.6.2
+* .NET Framework 4.8
+* .NET Standard 2.1
+* .NET 6.0
+* .NET 8.0
+* .NET 9.0
+
+## Installation {#installation}
+
+Install the package from NuGet:
+
+```bash
+dotnet add package ClickHouse.Driver
+```
+
+Or using the NuGet Package Manager:
+
+```bash
+Install-Package ClickHouse.Driver
+```
+
+## Usage {#usage}
+
+### Creating a Connection {#creating-a-connection}
+
+Create a connection using a connection string:
+
+```csharp
+using ClickHouse.Driver.ADO;
+
+var connectionString = "Host=localhost;Protocol=http;Database=default;Username=default;Password=";
+
+using (var connection = new ClickHouseConnection(connectionString))
+{
+    connection.Open();
+}
+```
+
+### Creating a Table {#creating-a-table}
+
+Create a table using standard SQL syntax:
+
+```csharp
```

Comment:
@Blargian what library do we use for code snippets? it doesn't seem to support c# syntax

Reply:
@mshustov fixed. If you need it in future you can add an additional language in `docusaurus.config.js` (https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages for reference)